imports/ui/components/settings/Settings.jsx

```diff
@@ -159,6 +164,16 @@ class SettingsPage extends Component {
         this.setState({[e.target.name]: val});
     }
 
+    updateProfile (){
+        name = this.state.name;
+        number = this.state.number;
+        email = this.state.email;
+        address = this.state.address;
+        username = this.state.username;
+        var info = {users: {name , number , email , address, username}};
```

Comment:
repeat calling `this.state` isn't good. There's a way to get the specific objects by key name in ES6. Search and implement that.